src/main/ts/commands/Publish.ts

```diff
@@ -43,7 +43,7 @@ const npmTag = async (args: PublishArgs, tags: string[], r: BranchDetails, dir:
     console.log('Setting NPM tags.');
     console.log('This will likely fail a few times, until the package is available in the registry - this is normal.');
     for (const t of tags) {
-      const fullPackageName = r.packageJson.name + '@' + Version.versionToString(r.version);
+      const fullPackageName = r.rootModule.packageJson.name + '@' + Version.versionToString(r.version);
```

Comment:
I'm confused by this PR as you only seem to tag the root module, what about the other modules? Or is this just an iterative step and you don't really support the other modules yet?

Reply:
It's just an iterative step.

Reply:
I'm trying to do small steps so the PRs are easier to review.